File: freecodecamp/daily-code-challenge/2026-01-19.py
JOULES_IN_CALORIE = 4184
JOULES_IN_WATTHOUR = 3600

def compare_energy(calories_burned, watt_hours_used):
    calories_burned_in_joules = calories_burned * JOULES_IN_CALORIE
    watt_hours_used_in_joules = watt_hours_used * JOULES_IN_WATTHOUR

    if (calories_burned_in_joules == watt_hours_used_in_joules):
        return "Equal"
    elif (calories_burned_in_joules > watt_hours_used_in_joules):
        return "Workout"
    else:
        return "Devices"

File: freecodecamp/daily-code-challenge/2026-01-19.js
const JOULES_IN_CALORIE = 4184;
const JOULES_IN_WATTHOUR = 3600;

function compareEnergy(caloriesBurned, wattHoursUsed) {
  const caloriesBurnedInJoules = caloriesBurned * JOULES_IN_CALORIE;
  const wattHoursUsedInJoules = wattHoursUsed * JOULES_IN_WATTHOUR;

  return (caloriesBurnedInJoules === wattHoursUsedInJoules 
    ? "Equal" 
    : caloriesBurnedInJoules > wattHoursUsedInJoules
      ? "Workout"
      : "Devices");
}
